### PR TITLE
Feature/#11 display processing errors

### PR DESCRIPTION
**Pull request by Skejven**
This code is for #11 - processing errors are now displayed on report tabs. 

---

I hereby agree to the terms of the AET Contributor License Agreement.

### CODE REVIEW

File: report/src/main/webapp/app/services/metadataEndpoint.service.js [hunk 1]
```diff
@@ -15,20 +15,20 @@
...
 	 */
-	function MetadataEndpointService($q, $http, configService, requestParametersService) {
+	function MetadataEndpointService($q, $http, endpointConfiguration, requestParametersService) {
 		var service = {
 				getMetadata: getMetadata,
 				saveMetadata: saveMetadata
 			},
 			requestParams = requestParametersService.get(),
-			configParams = configService.getConfig();
+			getEndpointUrl = endpointConfiguration.getEndpointUrl();
```
Comment (wiiitek) [inline report/src/main/webapp/app/services/metadataEndpoint.service.js]: Could we rename `getEndpointUrl` variable? To sth like
- `metadataUrl`
- `endpoint`
- ??

File: report/src/main/webapp/app/services/endpointConfiguration.service.js [hunk 2]
```diff
@@ -16,17 +16,16 @@
...
-				};
-
-				return config;
-			}
-		};
-	});
+  'use strict';
+  angularAMD.factory('endpointConfiguration', function () {
+    return {
+      getEndpointUrl: function () {
```
Comment (wiiitek) [inline report/src/main/webapp/app/services/endpointConfiguration.service.js]: As service is called `endpointConfiguration~maybe method could be renmd to `getUrl` ?
What do you think?

Comment (Skejven) [inline report/src/main/webapp/app/services/endpointConfiguration.service.js]: Yes, this is good idea - I renamed method to `getEndpoint` and `production` to `getUrl`.

**Comment by wiiitek** [comment]
For the source comparator there is a JavaScript error as Angular expects JSON object but artifactId is a HTML source. This should probably be investigated...

**Pull request merged**